test_files/preprocessor.h

```diff
@@ -0,0 +1,4 @@
+
+#define FOO 0
+#define FOOBAR 1
+#define FOOBARBAZ 2
```

Comment:
Current implementation blindly converts all defines to enum. e.g. if it is string it blindly converts it to enum FOO = "string".